Portfolio/index.html

```diff
@@ -6,8 +6,66 @@
     <link rel="stylesheet" href="index.css" />
```

Comment:
The style attribute in the body tag should have units specified for margin. For example, margin: 2px;